samples/bluetooth/rssi_nrf53/src/main.c

```diff
@@ -0,0 +1,130 @@
+/* main.c - Application main entry point */
+
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/types.h>
+#include <stddef.h>
+#include <sys/printk.h>
+#include <sys/util.h>
+
+#include <bluetooth/bluetooth.h>
+#include <bluetooth/hci.h>
+#include <bluetooth/hci_vs.h>
+
+struct rssi_wq_data {
+	struct k_work work;
+	bt_addr_le_t addr;
+	int8_t rssi;
+	uint8_t adv_type;
+} rssi_wq_data;
+
+static int8_t get_chip_temp(void)
+{
+	int err;
+	struct net_buf *rsp;
+	int8_t temperature;
+
+	err = bt_hci_cmd_send_sync(BT_HCI_OP_VS_READ_CHIP_TEMP, NULL, &rsp);
+	if (err) {
+		printk("HCI Error getting chip temp %d\n", err);
+		return -127;
+	}
+
+	struct bt_hci_rp_vs_read_chip_temp *cmd_data = (void *)rsp->data;
+
+	temperature = cmd_data->temps;
+	net_buf_unref(rsp);
+
+	return temperature;
+}
+
+static double pow(double x, double y)
+{
+	double result = 1;
+
+	if (y < 0) {
+		y = -y;
+		while (y--) {
+			result /= x;
+		}
+	} else {
+		while (y--) {
+			result *= x;
+		}
+	}
+
+	return result;
+}
+
+#define ROUND_FL_TO_UINT8(x) (uint8_t)((float)x + 0.5f)
+
+static void compensate_rssi(struct k_work *item)
```

Comment:
It would be nice if you could add a reference to the errata here

Reply:
I preferred having it in the README instead, but I can add the number at least and a ref to the readme